wiki/Universal_offset/en.md

```diff
@@ -2,12 +2,12 @@
 stub: true
 ---
 
-<!-- TODO: needs some explanation of the confusing negative/positive behavior and links to guide how to set it correctly -->
-
-# Universal offset
+# Universal Offset
 
 *For other uses, see [Offset (disambiguation)](/wiki/Disambiguation/Offset).*
 
-**Universal offset** is the audio offset [option](/wiki/Options) that applies to all songs in the osu! client. Setting it to a value other than `0` changes adds a delay between the currently playing song and the other sounds and graphics associated with it.
+**Universal Offset** is the audio offset [option](/wiki/Options) that applies to all [beatmaps](/wiki/Beatmap) in the osu! client. Setting it to a value other than `0` adds a delay between every beatmap's song and the other associated sounds and graphics. Setting the Universal Offset to a positive value will make the sounds and graphics associated with the beatmap appear earlier, while setting it to a negative value will make them appear later.
+
+Each player's optimal universal offset value differs due to variations in their systems. A Universal Offset, other than `0`, should only be used when **all beatmaps** have a noticeable delay, otherwise a [local offset](/wiki/Glossary/Local_song_offset) should be used instead. An incorrectly determined Universal Offset will cause significant timing problems on all beatmaps.
 
-Each player's optimal universal offset value differs due to variations in their systems. An incorrectly determined universal offset will cause significant timing problems on all [beatmaps](/wiki/Beatmap).
+Follow [this guide](/wiki/Guides/How_to_Use_the_Offset_Wizard) for step-by-step instructions on how to set your Universal Offset to the correct value using the [offset wizard](/wiki/Options/Offset_Wizard).
```

Comment:
This would be better formatted as a hatnote imo. Either way, "this guide" shouldn't be used as the title text and "your" should be removed

Reply:
Yeah, my bad. I'm not sure if i like it being a hatnote but if anyone else thinks it should be, ill change it :) thanks again